src/assets/translations/fr/main.json

```diff
@@ -1766,7 +1766,6 @@
         "secretRecoveryPhraseRequired": "La phrase secrète de récupération est requise",
         "secretRecoveryPhraseTooShort": "La phrase secrète de récupération est trop courte",
         "dragAndDrop": "Glisser-déposer ou Choisir un fichier",
-        "keystoreImportBody": "Téléchargez le fichier en cliquant sur Parcourir ou en le faisant glisser ci-dessous.",
```

Comment:
Is this expected?

Reply:
Yes, they are invalidated for the sync that starts on Monday.. a change occurred since the last translations. Thanks for the check!